packages/config/src/chainConfig/configs/mainnet.ts

```diff
@@ -97,6 +97,8 @@ export const chainConfig: ChainConfig = {
 
   // Networking
   // ---------------------------------------------------------------
+  // `MIN_VALIDATOR_WITHDRAWABILITY_DELAY + CHURN_LIMIT_QUOTIENT // 2` (= 33024, ~5 months)
+  MIN_EPOCHS_FOR_BLOCK_REQUESTS: 33024,
```

Comment:
Is this change related to the pruning?

Reply:
yes the node needs to keep blocks up to `MIN_EPOCHS_FOR_BLOCK_REQUESTS` to those can be served over p2p

> The minimum epoch range over which a node must serve blocks

Reply:
Indirectly related to pruning. This is a config defined by the consensus specs that defines the minimum window for serving blocks on the network.
A conformant implementation which stores the smallest amount of data to validate will derive their pruning strategy from this value.